.github/workflows/integration_test.yaml

```diff
@@ -9,11 +9,9 @@ jobs:
     secrets: inherit
     with:
       chaos-app-label: app.kubernetes.io/name=discourse-k8s
-      chaos-duration: 666
       chaos-enabled: true
       chaos-experiments: pod-delete
-      chaos-interval: 666
-      chaos-status-delay: 10
-      chaos-status-duration: 222
+      chaos-duration: 180
```

Comment:
Looks like we have a reduction in duration now, nice!

Reply:
Yes, the underlying issue was elsewhere and this duration gets us to the desired result :)